CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 
 - Updated README.md @ktsrivastava29
 - Added language to codeblocks in md files @ktsrivastava29
+- Updated simple.md @MdSahil-oss
```

Comment:
This still belongs under the "unreleased" section, not an already released version.